### ISSUE TITLE
Plugin fails to load when enabling

### ISSUE THREAD

**Issue by AriTheElk** [mokkiebear/heatmap-tracker#9]
I'm able to install, but once I click `Enable` I see a toast pop up that says "Failed to load plugin". In the dev console I see this:

```
Plugin failure: heatmap-tracker TypeError: Cannot read properties of null (reading 'colors')
    at Vu.loadSettings (plugin:heatmap-tracker:14:121367)
    at async Vu.onload (plugin:heatmap-tracker:14:120701)
```

**Comment by mokkiebear** [comment]
@AriTheElk thanks a lot!
Fixed in version 1.7.3

**Comment by AriTheElk** [comment]
Awesome, that worked! Thanks for the quick resolution 🙌
